var/spack/repos/builtin/packages/fmt/package.py

```diff
@@ -41,7 +41,7 @@ class Fmt(CMakePackage):
     variant(
         "cxxstd",
         default="11",
-        values=("98", "11", "14", "17", "20"),
+        values=("98", "11", "14", "17", "20", "23"),
```

Comment:
```suggestion
        values=("98", "11", "14", "17", "20", conditional("23", when="^cmake@3.20.3:")),
```
In theory CMake supports it from 3.20 onwards, but it very specifically does not yet work with 3.20.0-2.